randomizer/Lists/KasplatLocations.py

```diff
@@ -196,14 +196,18 @@ def setKasplat(self, state=True):
         KasplatLocation(
             name="Japes Kasplat: Up the Hill to the Painting Room",
             map_id=Maps.JungleJapes,
-            kong_lst=[Kongs.lanky, Kongs.tiny],
+            kong_lst=[Kongs.donkey, Kongs.diddy, Kongs.lanky, Kongs.tiny],
```

Comment:
Revert back to previous. See comment about Japes Underground Kasplat and fill failures